openqaoa/workflows/optimizer.py

```diff
@@ -75,9 +90,47 @@ def __init__(self, device=DeviceLocal('vectorized')):
         self.cloud_provider = list(DEVICE_ACCESS_OBJECT_MAPPER.keys())
         self.compiled = False
 
-    def asdict(self):
-        attributes_dict = convert2serialize(self)
-        return attributes_dict
+        # Initialize the identifier stamps, we initialize all the stamps needed to None
+        self.id = {
+            'uuid': None, #we generate a uuid for the experiment when we compile it
+            'parent_uuid': None, #user can set the parent uuid when they create the experiment using the set_identification method
+            'type': None, #type of the object
+            'datetime': datetime.now().strftime("%m/%d/%Y_%H:%M:%S") 
```

Comment:
Here we need to be careful. Let's make sure that ALL THE TIMES ARE IN UTC and the timezone is explicit recorded  by in the `strftime`. Also, better to change the order to the following `'%Y-%m-%dT%H:%M:%S.%f%z'`: the advantage is that they are all already time sorted (otherwise next to first of jan 2022 comes first of jan 2023 etc)

Reply:
I have now used `'%Y-%m-%d_%H:%M:%S.%f%z'` and `datetime.utcnow()` this way the time is always in utc+0, but I am not sure that you meant that. 
Was it that, or to use the local time and save somehow the time zone?

Reply:
No, that's fine :)